components/skus/browser/brave-rewards-cxx/Makefile

```diff
@@ -0,0 +1,21 @@
+CFLAGS := -DNOT_BRAVE_CORE_SHIM=1
```

Comment:
what is this for?

Reply:
I see you moved a similar Makefile under `src/` which generates the wrapper.h / wrapper.cc. This one can be used to build the example code. @evq will leave to you if we need it (see other makefile at `components/skus/browser/brave-rewards-cxx/src/Makefile `)